packages/navigator/src/browser/navigator-filter.ts

```diff
@@ -60,14 +64,10 @@ export class FileNavigatorFilter {
         this.emitter.fire(undefined);
     }
 
-    protected onPreferenceChanged(event: PreferenceChangeEvent<FileNavigatorConfiguration>): void {
-        let hasChanged = false;
+    protected onFilesPreferenceChanged(event: PreferenceChangeEvent<FileSystemConfiguration>): void {
```

Comment:
This cannot be renamed. Please restore the original method name. Sorry for not mentioning it before :(

Reply:
I left this method as it is, and added 
`protected onPreferenceChanged(event: PreferenceChangeEvent<FileNavigatorConfiguration>): void` with empty body. Please take another look at it :)